app/src/main/res/values/strings.xml

```diff
@@ -26,5 +26,6 @@
     <string name="scale_pref_summary">Scale for recording text</string>
     <string name="hello_world">Hello world!</string>
```

Comment:
Not actually part of this change, but I happened to notice this Hello world! string. I assume this will be presented as a string to be localized, and since it is not actually used, that would be confusing.